subprojects/gradle/lint-report/src/main/kotlin/com/avito/android/lint/LintReportTask.kt

```diff
@@ -45,10 +53,20 @@ abstract class LintReportTask : DefaultTask(), BuildVerdictTask {
         copyReports()
         val models = parseReports()
         sendToBitbucket(models)
+        if (slackReportChannel.isPresent) {
+            createLintSlackAlert().report(models, SlackChannel(slackReportChannel.get()))
+        }
         val report = mergeReports(models)
         failIfNeeded(models, report)
     }
 
+    private fun createLintSlackAlert(): LintSlackReporter {
+        return LintSlackReporter.Impl(
+            slackClient = slackClient.get(),
```

Comment:
check `isPresent` with readable exception

Reply:
i thought it's readable now by default; even with correct stacktrace
it's set in plugin right now, and i don't want to open PR just for that (is my exception message will be better than default?)

but i'm not sure and it's better be covered with even single test for whole project

Reply:
`Provider has no value: extension 'lintReport' property 'slackToken'`

yep, looks ok